python/lsst/summit/utils/butlerUtils.py

```diff
@@ -150,6 +158,75 @@ def makeDefaultLatissButler(
     return butler
 
 
+def makeDefaultButler(
+    instrument: str,
+    *,
+    extraCollections: list[str] | None = None,
+    writeable: bool = False,
+    embargo: bool = True,
+) -> dafButler.Butler:
+    """Create a butler for the instrument using default collections, regardless
+    of the location.
+
+    Parameters
+    ----------
+    extraCollections : `list` of `str`
+        Extra input collections to supply to the butler init.
+    writable : `bool`, optional
+        Whether to make a writable butler.
+    embargo : `bool`, optional
+        Use the embargo repo instead of the main one. Needed to access
+        embargoed data if not at a summit-like location.
+
+    Returns
+    -------
+    butler : `lsst.daf.butler.Butler`
+        The butler.
+    """
```

Comment:
You could add a `raises` section for the `FileNotFoundError` handling.

Reply:
I will add the `raises` section for good measure though.